<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRanchoWindow</class>
 <widget class="QWidget" name="addRanchoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 15px;
background-color: rgba(61, 255, 181,0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="10,90" columnstretch="90,10">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="logoutGroupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(76, 165, 34);
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
border-top-right-radius: 0px;

</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="logoutGroupBoxLayout" columnstretch="30,40,20">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <spacer name="horizontalSpacerLogout">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>192</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="titleLabel">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>30</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Agregar Nuevo Rancho</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="closeDialogButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(76, 165, 34);
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
border-top-left-radius: 0px;
border-top-right-radius: 25px;
}
QPushButton:hover{
	
	background-color: rgb(190, 67, 42);
}</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="logoutOptionsGroupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(216, 216, 216);
border-top-left-radius: 0px;
border-top-right-radius: 0px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="acceptDialogButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>880</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(116, 116, 116);
border-radius: 15px;
}
QPushButton:hover{
	
	background-color: rgb(76, 165, 34);
}</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelDialogButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>880</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(116, 116, 116);
border-radius: 15px;
}
QPushButton:hover{
background-color: rgb(190, 67, 42);
}</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>931</width>
        <height>831</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 40px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="addDireccionLineEdit">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>490</y>
         <width>531</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 3px solid;
border-radius: 10px;
border-color: rgb(156, 156, 156);
padding-left: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>Ej. Camino Rancho la Alegría </string>
       </property>
      </widget>
      <widget class="QLabel" name="estadoLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>300</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Estado:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="ranchoNameLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nombre del Rancho:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="paisLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>País:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="localidadLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>400</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Localidad:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="addRanchoLineEdit">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>100</y>
         <width>531</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 3px solid;
border-radius: 10px;
border-color: rgb(156, 156, 156);
padding-left: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>Ej. Rancho la Alegría</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="addPaisLineEdit">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>200</y>
         <width>531</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 3px solid;
border-radius: 10px;
border-color: rgb(156, 156, 156);
padding-left: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>30</number>
       </property>
       <property name="placeholderText">
        <string>Ej. México</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="addLocalidadLineEdit">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>400</y>
         <width>531</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 3px solid;
border-radius: 10px;
border-color: rgb(156, 156, 156);
padding-left: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>Ej. León de los Aldama</string>
       </property>
      </widget>
      <widget class="QLabel" name="direccionLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>490</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dirección:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="addEstadoLineEdit">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>300</y>
         <width>531</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 3px solid;
border-radius: 10px;
border-color: rgb(156, 156, 156);
padding-left: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>40</number>
       </property>
       <property name="placeholderText">
        <string>Ej. Guanajuato</string>
       </property>
      </widget>
      <widget class="QLabel" name="subtitleLabel">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>561</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>26</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ingrese los datos del Rancho</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="errorLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>770</y>
         <width>881</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 19, 19);</string>
       </property>
       <property name="text">
        <string>Invalid Data</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
      <widget class="QGroupBox" name="coordsGroupBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>570</y>
         <width>831</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
border: 3px solid;
border-radius: 10px;
border-color: rgb(156, 156, 156);
}
QGroupBox::title {
	background-color: rgba(255, 255, 255, 0);
padding-top: 5px;
padding-left: 5px;
border-bottom: 3px solid;
border-right: 3px solid;
border-bottom-right-radius: 10px;
border-color:rgb(156, 156, 156);
}
</string>
       </property>
       <property name="title">
        <string>Coordenadas</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="latitudLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>91</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string>Latitud</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="addLatitudLineEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>231</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 3px solid;
border-radius: 10px;
border-color: rgb(156, 156, 156);
padding-left: 5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="placeholderText">
         <string>Ej. 2447.08700</string>
        </property>
       </widget>
       <widget class="QLabel" name="longitudLabel">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>40</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string>Longitud:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="addLongitudLineEdit">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>40</y>
          <width>231</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 3px solid;
border-radius: 10px;
border-color: rgb(156, 156, 156);
padding-left: 5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="placeholderText">
         <string>Ej. 12100.52210</string>
        </property>
       </widget>
       <widget class="QPushButton" name="setCoordsButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>130</y>
          <width>461</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(156, 156, 156);
border-radius: 0px;
border-top-left-radius: 15px;
border-top-right-radius: 15px;
}
QPushButton:hover{
	background-color: rgb(20, 87, 105);
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Utilizar coordenadas actuales</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>closeDialogButton</tabstop>
  <tabstop>addRanchoLineEdit</tabstop>
  <tabstop>addPaisLineEdit</tabstop>
  <tabstop>addEstadoLineEdit</tabstop>
  <tabstop>addLocalidadLineEdit</tabstop>
  <tabstop>addDireccionLineEdit</tabstop>
  <tabstop>cancelDialogButton</tabstop>
  <tabstop>acceptDialogButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeDialogButton</sender>
   <signal>clicked()</signal>
   <receiver>addRanchoWindow</receiver>
   <slot>closeDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>978</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelDialogButton</sender>
   <signal>clicked()</signal>
   <receiver>addRanchoWindow</receiver>
   <slot>closeDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>988</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptDialogButton</sender>
   <signal>clicked()</signal>
   <receiver>addRanchoWindow</receiver>
   <slot>doDialogAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>988</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setCoordsButton</sender>
   <signal>clicked()</signal>
   <receiver>addRanchoWindow</receiver>
   <slot>setCoordinates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>866</y>
    </hint>
    <hint type="destinationlabel">
     <x>998</x>
     <y>885</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doDialogAction()</slot>
  <slot>closeDialog()</slot>
  <slot>setCoordinates()</slot>
 </slots>
</ui>
